qcengine/programs/terachem.py

```diff
@@ -172,12 +150,20 @@ def parse_output(self, outfiles: Dict[str, str], input_model: 'ResultInput') ->
                        gradients.append( float(x) )
 
         # Look for the last line that is the SCF info 
+        DECIMAL = r"""(
+          (?:[-+]?\d*\.\d+(?:[DdEe][-+]?\d+)?) |  # .num with optional sign, exponent, wholenum
+          (?:[-+]?\d+\.\d*(?:[DdEe][-+]?\d+)?)    # num. with optional sign, exponent, decimals
+        )"""
+
```

Comment:
feel free to `from qcel.molparse.regex import NUMBER` and use it directly

Reply:
Hi Lori, I didn't directly use the NUMBER definition of `qcel.molparse.regex` because it includes integer numbers, which I would like to exclude here. TeraChem prints FON numbers in between the SCF lines with format "int decimal int decimal int decimal", and real SCF lines are like "int decimal decimal decimal decimal decimal", and I would like to distinguish them.

Reply:
makes sense, tks. perils of parsing that we're one `.` away from grabbing wrong number.

Reply:
@loriab would it be good to put more regex values in QCEl?

Reply:
probably so. I found a place in pubchem where I needed the equivalent of `DECIMAL`. More than once or twice and should probably be added to molparse.regex (in string form of course so that can be composed).